### PR TITLE
DOC: update the pandas.Series.dropna docstring by Ankara Chapter

### PR DESCRIPTION
**Pull request by mselimata**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [ x] PR title is "DOC: update the <your-function-or-method> docstring"
- [x ] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [ x] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ x] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [x ] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
# paste output of "scripts/validate_docstrings.py <your-function-or-method>" here
# between the "```" (remove this comment, but keep the "```")

```

If the validation script still gives errors, but you think there is a good reason
to deviate in this case (and there are certainly such cases), please state this
explicitly.

Checklist for other PRs (remove this part if you are doing a PR for the pandas documentation sprint):

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Hmm can you check to see if your editor changed the newline mode of the document? The diff should just be the lines you changed, not the entire file.

Ping me in the chat if you need help figuring it out.

**Comment by Neuroscitechie** [comment]
I collected edits from everyone and uploaded the file instead of updating the lines only

**Review by jreback — CHANGES_REQUESTED** [review]
something when wrong here as the whole file is changed.

File: pandas/core/series.py [hunk 1]
```diff
@@ -2970,13 +2970,49 @@ def notnull(self):
 
     def dropna(self, axis=0, inplace=False, **kwargs):
         """
-        Return Series without null values
+        Return Series without null values.
+
+        In case of missing data, any or all values can be used reciprocally.
```
Comment (TomAugspurger) [inline pandas/core/series.py]: Not sure what you mean by "reciprocally". I think this can section can be removed.

File: pandas/core/series.py [hunk 2]
```diff
@@ -2970,13 +2970,49 @@ def notnull(self):
...
+        All of missing inputs will be credited.
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns'}, or tuple/list thereof
+            Pass tuple or list to drop on multiple axes.
```
Comment (TomAugspurger) [inline pandas/core/series.py]: This method is series-specific, so note that \`axis\`  is included for compatability with `DataFrame.dropna`.

File: pandas/core/series.py [hunk 3]
```diff
@@ -2970,13 +2970,49 @@ def notnull(self):
...
+        inplace : boolean, default False
+            If True, do operation inplace and return None.
+        kwargs : {'any', 'all'}
```
Comment (TomAugspurger) [inline pandas/core/series.py]: Format as

```
**kwargs
    For compatibility with `DataFrame.dropna`. No additional arguments are used.
```

File: pandas/core/series.py [hunk 4]
```diff
@@ -2970,13 +2970,49 @@ def notnull(self):
...
 
         Returns
         -------
         valid : Series
         inplace : boolean, default False
             Do operation in place.
+
+        See Also
+        --------
+        core.series._sanitize_array : To understand why is array one-dimensional.
```
Comment (TomAugspurger) [inline pandas/core/series.py]: Remove this, it's a private method.

Add

```
pandas.isna : Top-level NA detection method.
Series.isna : Detect missing values.
Series.fillna : Fill missing values.
DataFrame.dropna : Same method for DataFrames.
```

File: pandas/core/series.py [hunk 5]
```diff
@@ -2970,13 +2970,49 @@ def notnull(self):
...
+
+        Examples
+        --------
+        >>> s = pd.Series([None,2,None,0,3])
```
Comment (TomAugspurger) [inline pandas/core/series.py]: PEP8: space after comma.

File: pandas/core/series.py [hunk 6]
```diff
@@ -2970,13 +2970,49 @@ def notnull(self):
...
+        >>> s.head()
+        0    NaN
+        1    2.0
+        2    NaN
+        3    0.0
+        4    3.0
+        dtype: float64
+
+        To omit these NaN values, use dropna() function
```
Comment (TomAugspurger) [inline pandas/core/series.py]: Can remove this. Just show the example.

File: pandas/core/series.py [hunk 7]
```diff
@@ -2970,13 +2970,49 @@ def notnull(self):
...
+
+        >>> s.dropna(axis=0, inplace=False, how='any')
```
Comment (TomAugspurger) [inline pandas/core/series.py]: Remove False and `how='any'`. Just show the defaults.

File: pandas/core/series.py [hunk 8]
```diff
@@ -2970,13 +2970,49 @@ def notnull(self):
...
+        1    2.0
+        3    0.0
+        4    3.0
+        dtype: float64
         """
```
Comment (TomAugspurger) [inline pandas/core/series.py]: Show that `s` is unmodified and then show an example with `inplace=True` that returns None and modifies `s` inplace.

**Review by TomAugspurger — COMMENTED** [review]
Pushed a commit fixing the line endings.

@Neuroscitechie there may be a setting in your editor handling line endings. We want `LF`, not `CRLF`.

https://help.github.com/articles/dealing-with-line-endings/#platform-windows

**Comment by jorisvandenbossche** [comment]
We seem to have a conflicting PR: https://github.com/pandas-dev/pandas/pull/20204 (that one is doing both DataFrame and Series dropna)

**Comment by TomAugspurger** [comment]
Closing in favor of #20204.

**Pull request closed without merging**